ROMFS/px4fmu_common/init.d-posix/px4-rc.simulator

```diff
@@ -86,10 +86,6 @@ elif [ "$PX4_SIMULATOR" = "gz" ] || [ "$(param show -q SIM_GZ_EN)" -eq "1" ]; th
 
 		# start gz bridge with pose arg.
 		if gz_bridge start -p "${model_pose}" -m "${PX4_GZ_MODEL}" -w "${PX4_GZ_WORLD}" -i "${px4_instance}"; then
-			if param compare -s SENS_EN_BAROSIM 1
-			then
-				sensor_baro_sim start
-			fi
```

Comment:
I think this should stay, but the sensor sim should be disabled in the airframe config. Set  the parameter to zero
https://github.com/PX4/PX4-Autopilot/blob/b66e15c4b967a144b193d6b71f4f0b315e7631f0/ROMFS/px4fmu_common/init.d-posix/airframes/4001_gz_x500#L17

Reply:
https://github.com/PX4/PX4-Autopilot/pull/21176/commits/e0de6492a0bb545b661adcd0234d21856aa25c26